.github/workflows/build_and_test.yml

```diff
@@ -10861,6 +11355,33 @@ jobs:
         cd OpenDDS/tests/cmake/build
         cmake -DCMAKE_CXX_STANDARD=11 ..
         cmake --build . -- -j4
+    - name: set up TAO Hello test
+      shell: bash
+      run: |
+        cd OpenDDS
+        . setenv.sh
+        mwc.pl -type gnuace -workers 4 $TAO_ROOT/tests/Hello
+    - name: build TAO Hello test
+      shell: bash
+      run: |
+        cd OpenDDS
+        . setenv.sh
+        make -j4 dir=$TAO_ROOT/tests/Hello
+    - name: set up Modeling tests
```

Comment:
The TAO Hello test and modeling tests are also built here but it seems to me they are not run in any test job.

Reply:
I'm OK with this. We might need a couple followup tasks to flesh out and fix MacOS testing in CI. This task is just to get _something_ back up and running.